src/os/interaction/interaction.js

```diff
@@ -91,7 +91,10 @@ os.interaction.configureCesium = function(sscc) {
   sscc.rotateEventTypes = [Cesium.CameraEventType.LEFT_DRAG];
 
   // mouse wheel without modifiers to zoom
-  sscc.zoomEventTypes = [Cesium.CameraEventType.WHEEL];
+  sscc.zoomEventTypes = [Cesium.CameraEventType.WHEEL, Cesium.CameraEventType.PINCH, {
```

Comment:
Pinch zoom on a trackpad (at least on a MacBook Pro) fires a wheel event with a Ctrl modifier. It does not use pinch events.